buildSrc/src/main/groovy/com/github/okbuilds/okbuck/rule/GenSqlDelightRule.groovy

```diff
@@ -0,0 +1,30 @@
+package com.github.okbuilds.okbuck.rule
+
+final class GenSqlDelightRule extends BuckRule {
+
+    private final String mSqlDelightRunnerPath
+    private final String mProjectDir
+
+    GenSqlDelightRule(String name, String sqlDelightRunnerPath, String projectDir) {
+        super("gen", name)
+        mSqlDelightRunnerPath = sqlDelightRunnerPath
+        mProjectDir = projectDir
+    }
+
+    @Override
+    final void print(PrintStream printer) {
+        printer.println("genrule(")
+        printer.println("\tname = '${name}',")
+        printer.println("\tsrcs = glob([")
+        printer.println("\t\t'src/main/sqldelight/**/*.sq',")
```

Comment:
why just src/main? can you just iterate over the srcDirs instead of this target?